src/main/java/core/basesyntax/servce/impl/Reader.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.servce.impl;
+
+import com.opencsv.CSVReader;
+import com.opencsv.CSVReaderBuilder;
+import com.opencsv.exceptions.CsvException;
+import core.basesyntax.servce.CsvFileReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.List;
+
+public class Reader implements CsvFileReader {
```

Comment:
```suggestion
public class CsvFileReader implements FileReader {
```
FileReader - abstraction
CsvFileReader - implementation